contrib/src/test/java/org/apache/apex/malhar/contrib/parser/LogParserTest.java

```diff
@@ -18,16 +18,16 @@
  */
```

Comment:
There is no functional change in this file. It seems that your IDE has adjusted the whitespaces... Please revert this file to master.